src/frequently-requested-changes.md

```diff
@@ -0,0 +1,164 @@
+languages. Arguments range from the ergonomic ("I don't want to type this") to
+the aesthetic ("I don't like how this looks").
+
+Changes that would break existing Rust code are non-starters. Even in an
+edition, changes this fundamental remain extremely unlikely. The established
+Rust community with knowledge of existing Rust syntax has a great deal of
+value, and to be considered, a syntax change proposal would have to be not just
+*better*, but *so wildly better* as to overcome the massive downside of
+switching.
+
+In addition, such changes often go against one or more other aspects of Rust's
+design philosophy. For instance, we don't want to make changes that make code
+easier to write but harder to read, or changes that make code more error-prone
+to modify and maintain.
+
+That said, *we are open to proposals that involve new syntax*, especially for
+new features, or to improve an existing fundamental feature. The bar for new
+syntax (e.g. new operators) is high, but not insurmountable. But the bar for
+*changes to existing syntax* is even higher.
+
+## Changes to avoid writing `self.method()` when calling a method from a method
+
+In Rust, within a method of an object, calling another method requires writing
+`self.othermethod()`, and accessing a field requires `self.field`. In some
+other languages, such accesses can omit the equivalent of `self.`, and just
+write `othermethod` or `field`, implicitly referencing the "current object".
+People writing code with such method calls or field accesses sometimes ask for
+such shorthand in Rust.
+
+Rust prefers the explicitness of writing `self.` for method calls and field
+accesses, and in most ways treating `self` as a normal object of the type. This
+avoids unexpected calls to the wrong method, makes it easier to distinguish
+methods and free functions, and makes code easier to read. We don't want to
+provide a shorthand that makes this syntax briefer at the expense of this
+clarity and unambiguity.
+
+## Arbitrary custom operator syntax
+
+Rust allows overloading existing operators; for instance, you can implement the
+`Add` trait to overload the `+` operator. However, Rust does not allow creating
+*new* operators.
+
+Some would argue this can make code more readable, if you know what the
+operators mean. If you don't, it makes the code inscrutable.
+
+In general, such a change would substantially raise barriers to entry for Rust
+developers, making Rust code less approachable and less universally
+understandable. We're unlikely to add support for this.
+
+Note that Rust's existing operator overloading is semantic (`impl Add`), not
+syntactic (`impl Plus`), which tends to encourage using overloaded operators
+for the same semantic purposes, rather than for building arbitrary domain-specific languages.
+
+Rust developers seeking to build arbitrary domain-specific languages (DSLs)
+should consider the macro system.
+
+## Numeric overflow checking should be on by default even in release mode
```

Comment:
This one feels qualitatively different from basically everything else, since it's a "we want to change it, if only the perf was tolerable", rather than a "yeah, almost certainly no".  I don't disagree with the text, though, so it doesn't need to change, I guess.

Reply:
That's fair. I think it still fits, and I'll cheer if someone shows up with the benchmark numbers letting us change it. I agree that it's the *closest* on the list to being changeable.